manifests-maistra/2.1.0/servicemeshcontrolplanes.crd.yaml

```diff
@@ -4967,6 +4967,8 @@ spec:
                     type: object
                   dataPlane:
                     properties:
+                      automaticNetworkPolicy:
```

Comment:
nit, maybe `configureMeshNetworkPolicyRules` or enable, or install, or manage?

Reply:
Thanks. Naming things is hard.
@dgn @luksa Any [other] suggestion here?

Also, I've put this under `spec.security.dataPlane`. Does that sound good?

Reply:
That seems like the correct place to me.